Simlar/src/SMLRLinphoneHandler.m

```diff
@@ -739,19 +739,16 @@ - (void)registrationStateChanged:(const LinphoneProxyConfig *const)cfg state:(co
     }
 }
 
-- (BOOL)callEnded:(const LinphoneCallState)state
+- (LinphoneCallState)fixLinphoneCallState:(const LinphoneCallState)state
```

Comment:
This method does not fix the LinphoneCallState. `callStateChanged` does that it using the fixed state. So, this method would be better named `fixedLinphoneCallState`.